src/Features/VisualBasic/Portable/InitializeParameter/InitializeParameterHelpers.vb

```diff
@@ -15,9 +15,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.InitializeParameter
 
         Public Shared Function GetBlockOperation(functionDeclaration As SyntaxNode, semanticModel As SemanticModel, cancellationToken As CancellationToken) As IBlockOperation
```

Comment:
Should the signature of this method also be changed to follow `TryGet` pattern as it can return `Nothing`?